### PR TITLE
Remove Stack Overflow and Spectrum from project and use GH Discussions

### PR DESCRIPTION
**Pull request by derrickmehaffy**
#### Description of what you did:

This PR removes all references to Stack Overflow and Spectrum Chat in favor of using the Github Discussions.

### CODE REVIEW

**Comment by abdonrd** [comment]
I also want to replace the issues template with the link:

https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

Real example:

https://github.com/zeit/next.js/issues/new/choose

**Comment by derrickmehaffy** [comment]
Thank you @abdonrd I changed it to have a link.

**Comment by lauriejim** [comment]
I think we also have to remove this file - https://github.com/strapi/strapi/blob/master/.github/ISSUE_TEMPLATE.md
Since blank issues are not allowed.

File: packages/strapi-admin/admin/src/containers/HomePage/SocialLink.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ import PropTypes from 'prop-types';
 import Gh from '../../assets/images/social_gh.png';
 import Slack from '../../assets/images/social_slack.png';
 import Medium from '../../assets/images/social_medium.png';
-import So from '../../assets/images/social_so.png';
```
Comment (soupette) [inline packages/strapi-admin/admin/src/containers/HomePage/SocialLink.js:12]: Can you remove the asset?

**Pull request merged**